python/nav/django/templatetags/info.py

```diff
@@ -164,3 +164,9 @@ def sortdict(dictionary, reverse=False):
 def is_list(value):
     """Returns True if the value is a list"""
     return isinstance(value, list)
+
+
+@register.filter
+def dunderless(value):
```

Comment:
This should also have a docstring that explains the function and its usage. Pylint complain about this, but I guess most new violations here are drowned out by the existing mass of violations ;-)